js/apps/admin-ui/src/keycloak.ts

```diff
@@ -9,6 +9,7 @@ export const keycloak = new Keycloak({
 });
 
 export async function initKeycloak() {
+  keycloak.onAuthLogout = () => keycloak.login();
```

Comment:
This callback can be applied directly after constructing the Keycloak instance, no need to wait until initialization.

Reply:
No because the return type of onAuthLogout is void

Reply:
Not according the documentation

Reply:
> The adapter supports setting callback listeners for certain events. Keep in mind that these have to be set before the call to the `init()` method.

So you can still set this outside of the `initKeycloak()` method, right after constructing the Keycloak instance.